### PR TITLE
Add nighlty run for remote-store and 1-replica cluster config

### PR DESCRIPTION
**Pull request by rishabh6788**
### Description
This PR: 
1. Adds support to run nightly benchmarks on clusters with 2-replica and 1-replica separately. 
2. Adds support to run nightly-benchmark for `Remote Store` feature set to be released in 2.9. 

### Issues Resolved
https://github.com/opensearch-project/OpenSearch/issues/8087 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: jenkins/opensearch/benchmark-test.jenkinsfile [hunk 1]
```diff
@@ -70,6 +106,11 @@ pipeline {
             description: 'Use OpenSearch min distribution',
             defaultValue: false
         )
+        booleanParam(
+            name: 'ENABLE_REMOTE_STORE',
```
Comment (peterzhuamazon) [inline jenkins/opensearch/benchmark-test.jenkinsfile:110]: This should be setup in cron as well:
`ENABLE_REMOTE_STORE`

Comment (rishabh6788) [reply jenkins/opensearch/benchmark-test.jenkinsfile:110]: Added.

**Review by peterzhuamazon — APPROVED** [review]
Approve, tho this can definitely be reduced in length with a env var in Jenkinsfile.

Suggest improvement later.

Thanks.

**Comment by ankitkala** [comment]
@rishabh6788 Can you confirm that SegRep experimental flag `opensearch.experimental.feature.segment_replication_experimental.enabled` is enabled for this run? Segment Replication integration with remote store is behind this flag.

**Comment by rishabh6788** [comment]
These are the settings added to nightly remote-store runs. 

```
opensearch.experimental.feature.remote_store.enabled:true 
cluster.remote_store.enabled:true 
opensearch.experimental.feature.segment_replication_experimental.enabled:true. 
cluster.indices.replication.strategy:SEGMENT
```
@ankitkala

**Pull request merged**